CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 - Handles cases where certain control flow type expressions have comments between patterns/keywords and the pattern ident contains the keyword [#5009](https://github.com/rust-lang/rustfmt/issues/5009)
 - Handles tuple structs that have explicit visibilities and start with a block style comment [#5011](https://github.com/rust-lang/rustfmt/issues/5011)
 - Handles leading line-style comments in certain types of macro calls [#4615](https://github.com/rust-lang/rustfmt/issues/4615)
+- bizarre string indentation in macro defs with `format_strings` [#4036](https://github.com/rust-lang/rustfmt/issues/4036)
```

Comment:
Did you mean to write "Fixes bizarre string ..."?

Reply:
Good catch